scripts/us_census/acs5yr/subject_tables/common/generate_col_map.py

```diff
@@ -0,0 +1,141 @@
+"""Module to generate a column map to their corresponding statistical variable defintions."""
+
+import os
+import json
+
+import pandas as pd
+
+#TODO:Remove this import and make it from ~/data/utils/ directory when it is available
+os.system(
+    'curl https://raw.githubusercontent.com/Abilityguy/data/dcid_gen/util/statvar_dcid_generator.py -LO statvar_dcid_generator.py'
+)
+from statvar_dcid_generator import get_stat_var_dcid
+
+
+
+def generate(specDict: dict, columnList: list) -> dict:
+    """Wrapper function for generateColMapBase class to generate column map.
+
+  Args:
+    specDict:
+      A dictionary containing specifications for the different properties of the statistical variable.
+    columnList:
+      A list of column names for which the column map needs to be generated. This is typically the column header in the dataset.
+
+  Returns:
+    A dict mapping each column to their respective stat_var node definitions.
+  """
+    col_map_obj = GenerateColMapBase(specDict=specDict, columnList=columnList)
+    return col_map_obj._generate_stat_vars_from_spec()
+
+
+class GenerateColMapBase:
+    """module to generate a column map given a list of columns of the dataset and a JSON Spec
+
+  Attributes:
+    specDict: A dictionary containing specifications for the different properties of the statistical variable.
+    columnList: A list of column names for which the column map needs to be generated. This is typically the column header in the dataset.
+  """
+
+    def __init__(self, specDict=None, columnList=None):
+        """module init """
+        self.features = specDict
+        self.column_list = columnList
+        self.column_map = {}
+
+    def _generate_stat_vars_from_spec(self):
+        """generates stat_var nodes for each column in column list and returns a dictionary of columns with their respective stat_var nodes"""
```

Comment:
please wrap comment lines at 80 columns. Not sure if yapf does this automatically.

Reply:
Before pushing the code, I ran yapf as well pylint. I couldn't get the comments, specifically docstrings wrapped to 80 columns. Can it be done manually?

Reply:
Can yo also describe the structure of stat var node returned?
Is it a string or a list, or dictionary of property values?